### ISSUE TITLE
Image with EXIF data will crash directus

### ISSUE THREAD

**Issue by henk23** [directus/directus#16971]
### Describe the Bug

Similar to #7206: Uploading a certain image (containing EXIF data) will cause the directus process to crash.

### To Reproduce

Upload this image to directus via the File Library.

![kataster](https://user-images.githubusercontent.com/10068060/210128423-6cc89c0c-3306-4577-96f2-bd89a55153a0.png)

### Errors Shown

```
2022-12-31 08:02:06 /directus/node_modules/exif-reader/index.js:5
2022-12-31 08:02:06     throw new Error('Invalid EXIF data: buffer should start with "Exif".');
2022-12-31 08:02:06           ^
2022-12-31 08:02:06 
2022-12-31 08:02:06 Error: Invalid EXIF data: buffer should start with "Exif".
2022-12-31 08:02:06     at module.exports (/directus/node_modules/exif-reader/index.js:5:11)
2022-12-31 08:02:06     at Object.<anonymous> (/directus/node_modules/directus/dist/services/files.js:152:103)
2022-12-31 08:02:06 
2022-12-31 08:02:06 Node.js v18.12.1
```

### What version of Directus are you using?

9.22.1

### What version of Node.js are you using?

18.12.1

### What database are you using?

SQLite 3.39.2

### What browser are you using?

Firefox

### How are you deploying Directus?

Docker

**Comment by Kinzi** [comment]
I'm getting the same!

**Comment by Kinzi** [comment]
I believe there is a new version of the underlying npm package fixing this problem:

https://github.com/devongovett/exif-reader/tags
https://github.com/devongovett/exif-reader/pull/21

**Comment by Kinzi** [comment]
Apparently this is already patched but not yet released:
https://github.com/directus/directus/commit/40f94e6907d2999341546838720cf3086653515b

**Comment by azrikahar** [comment]
@Kinzi great detective work! I can confirm the provided image does cause error on `9.22.1` (exif-reader `1.0.3`), but it is indeed resolved on main branch (exif-reader `1.1.0`) as you pointed out. 

Marking this as resolved by #16875.

@henk23 thank you very much for providing the image to reproduce this 👍

**Comment by Kinzi** [comment]
@azrikahar when is this released though? ;)

**Comment by azrikahar** [comment]
> @azrikahar when is this released though? ;)

I'm not aware of an exact release date at the moment, but we are currently in the process of reassessing the general release strategy and cadence.

I do believe #16922 and #16942 are planned to be included in the upcoming release, so a tentative release date would be after they both are merged :)

**Comment by c-nv-s** [comment]
just ran into this issue too :-( keeping an eye out for next release date/estimate.

**Comment by Kinzi** [comment]
@azrikahar any chance to speed up the release here? This is causing some serious user annoyance for us unfortunately....

**Comment by azrikahar** [comment]
@Kinzi I'm not sure would I be able to further expedite the release here, but would you mind trying [npm overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json?v=true#overrides) (or yarn resolutions, pnpm overrides depending on the package manager you are using) in the meantime? An example of steps to take would be:

1. (optional) `npm why exif-reader` just to confirm it's currently version `1.0.3` for Directus 9.22.1
2. Add the following in package.json (if you are using npm):

    ```json
    "overrides": {
        "exif-reader": "1.1.0"
    }
    ```
    
3. `npm install` to update the version
4. (optional) `npm why exif-reader` again so that we can confirm it is now `1.1.0`

**Comment by Kinzi** [comment]
@azrikahar will do thanks. Didn't know about that!

**Comment by Kinzi** [comment]
Not working :(

also tried to remove node_modules and install fresh...

**Comment by azrikahar** [comment]
@Kinzi what version of npm are you currently on? I believe "overrides" was introduced in npm `8.3.0` based on npm/cli#4150. According to https://nodejs.org/en/download/releases/, npm `8.3.0` should be included in Node `16.14.0` and above.

**Comment by Kinzi** [comment]
@azrikahar goood catch! I was just below that version :D

This worked, thanks!
